src/games/strategy/triplea/ui/TripleAFrame.java

```diff
@@ -1481,22 +1431,7 @@ public TechRoll getTechRolls(final PlayerID id) {
     m_messageAndDialogThreadPool.waitForAll();
     m_actionButtons.changeToTech(id);
     // workaround for panel not receiving focus at beginning of tech phase
-    if (!SwingUtilities.isEventDispatchThread()) {
-      try {
-        SwingUtilities.invokeAndWait(new Runnable() {
-          @Override
-          public void run() {
-            requestFocusInWindow();
-            transferFocus();
-          }
-        });
-      } catch (final Exception e) {
-        e.printStackTrace();
-      }
-    } else {
-      requestFocusInWindow();
-      transferFocus();
-    }
+    requestWindowFocus();
```

Comment:
Do we not need the invokeAndWait here? Also looks like transferFocus() was removed?